core/domain/subtopic_page_domain_test.py

```diff
@@ -98,33 +98,28 @@ def test_create_default_subtopic_page(self):
         }
         self.assertEqual(subtopic_page.to_dict(), expected_subtopic_page_dict)
 
-    def test_get_subtopic_page_id(self):
+    def test_get_subtopic_page_id(self) -> None:
         self.assertEqual(
             subtopic_page_domain.SubtopicPage.get_subtopic_page_id('abc', 1),
             'abc-1')
 
-    def test_get_subtopic_id_from_subtopic_page_id(self):
+    def test_get_subtopic_id_from_subtopic_page_id(self) -> None:
         self.assertEqual(
             self.subtopic_page.get_subtopic_id_from_subtopic_page_id(), 1)
 
-    def _assert_validation_error(self, expected_error_substring):
+    # We have ignored [override] here because Signature of
+    # "_assert_validation_error" is incompatible with supertype "TestBase"
+    # https://mypy.readthedocs.io/en/stable/error_code_list.html#check-validity-of-overrides-override
```

Comment:
```suggestion
    # We have ignored [override] here because the signature of this method
    # doesn't match with TestBase._assert_validation_error().
```

Reply:
Done!